digiroad2-oracle/src/main/resources/db/migration/V0_13__create_nodes_and_junctions.sql

```diff
@@ -0,0 +1,109 @@
+  CREATE TABLE ROADWAY_POINT
+   (	ID NUMBER NOT NULL ENABLE,
```

Comment:
No need "ENABLE"

Reply:
I retrieve ddls straight from sql developer. Should I delete 'enable' and index configs?

Reply:
Yes, I prefer keeping these simple.

Reply:
Done